ports/servoshell/desktop/cli.rs

```diff
@@ -103,10 +106,23 @@ pub fn main() {
     let event_loop = EventsLoop::new(opts::get().headless, opts::get().output_file.is_some())
         .expect("Failed to create events loop");
 
+    // Implements window methods, used by compositor.
+    let window = if opts::get().headless {
+        // GL video rendering is not supported on headless windows.
+        set_pref!(media.glvideo.enabled, false);
```

Comment:
It would be nice to log a warning here if `media.glvideo.enabled` was previously `true`